src/event_funcs.c

```diff
@@ -0,0 +1,29 @@
+#include "global.h"
+
+#include "unit.h"
+
+extern struct Unit * data_021974d8;
+
+// FE11U = 0x02044974
+bool UnitSetEventDead(void * unused, s32 unitId)
+{
+
+    struct Unit * unit;
+    
+    if (!unitId)
+    {
+        unit = 0;
```

Comment:
As a very minor style nitpick, I'd generally use the `NULL` macro if we're setting a pointer value to null, but it's not critical by any means